readme.md

```diff
@@ -54,9 +68,16 @@ console.log(results);
 
 Your Answer:
 ```text
-
+The .find() method requires a callback function, so it would look more like the below (or just skip the results variable and console.log(author)).
+```
+```js
+AuthorModel.find({name: "Bob"}, function(err, author){
+  var results = author;
+  console.log(results);
+}
```

Comment:
:+1:
Using `author` makes sense to me.  Why go from a specific name to a non-specific name?